src/components/util/Count.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect, useRef } from 'react';
+
+const Count: React.FC = () => {
+  const [data, setData] = useState<any>([]);
+  const [count, setCount] = useState(0);
+  const [loading, setLoading] = useState(true);
+  const intervalRef: any = useRef();
+
+  useEffect(() => {
+    fetchData();
+    intervalRef.current = setInterval(() => {
+      setCount(count + 1);
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/taulant-eulerity/Canva/pull/4#issuecomment-2181154623>#57299f</a><span> - 06/20/2024, 05:05 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The 'setCount' function inside the 'setInterval' callback is using the 'count' state directly, which can lead to stale state issues because the 'count' value might not be the latest.
</div>
<div id="fix">
<b>Fix:</b> Use the functional form of 'setCount' to ensure the latest state is used.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -11,7 +11,7 @@
     intervalRef.current = setInterval(() => {
 -      setCount(count + 1);
 +      setCount(prevCount => prevCount + 1);
     }, 1000);
```
</div>
</details>
</div>
<br>


</div>